### PR TITLE
Get bot online

### PR DESCRIPTION
**Pull request by vinceTheProgrammer**
Some basic code I wrote to get the bot online. My first ever pull request. Not sure if my code follows best practices. Feel free to optimize/clean up the code.

P.S. I just realized that I probably should have used the development branch instead of creating a new one 🤦‍♂️

### CODE REVIEW

**Comment by vinceTheProgrammer** [comment]
I'm not sure if we need `.gradle/` and `build/`

There were merge conflicts, so I added those directories to the .gitignore and manually did `git rm` on these files:

- .gradle/6.7/executionHistory/executionHistory.bin
- .gradle/6.7/executionHistory/executionHistory.lock
- .gradle/6.7/fileHashes/fileHashes.bin
- .gradle/6.7/fileHashes/fileHashes.lock
- .gradle/buildOutputCleanup/buildOutputCleanup.lock
- .gradle/buildOutputCleanup/outputFiles.bin

**Review by JacobDevelopment — CHANGES_REQUESTED** [review]
Good, just need to change some minor things then we can push it. Adding those to the .gitignore is good, I forgot so no issues there.

File: src/main/java/io/jking/coursebot/Config.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import net.dv8tion.jda.api.utils.data.DataObject;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+
+public class Config {
+
+    public DataObject configObject;
```
[the remark this answers is not in the document]
Comment (vinceTheProgrammer) [reply src/main/java/io/jking/coursebot/Config.java:11]: Oh yeah, I forgot about that. This is good feedback for me in learning good practices in Java. I'll fix that right away.

File: src/main/java/io/jking/coursebot/Starter.java [hunk 2]
```diff
@@ -1,9 +1,32 @@
...
+import java.io.IOException;
+
 public class Starter {
 
-    public static void main(String[] args) {
+    public static void main(String[] args) throws LoginException, InterruptedException, IOException {
         System.out.println("Hello World!");
+
+        final Config config = new Config("config.json");
+        String token = (String) config.configObject.get("token");
```
[the remark this answers is not in the document]
Comment (vinceTheProgrammer) [reply src/main/java/io/jking/coursebot/Starter.java:18]: Fair enough, I'll make the necessary changes.

**Comment by vinceTheProgrammer** [comment]
I probably could have merged development into get-bot-online more efficiently, but I think I figured out how to merge everything correctly. You might still want to check it to make sure though.

**Pull request merged**